### DISCUSSION TITLE
Invest Accounts Do Not Show On Dashboard

### DISCUSSION THREAD

**Discussion by Ocirasai** [moneymanagerex/moneymanagerex#6162] [Q&A] [answered]
Hello:

I created a few accounts as type **Investment**. After creating them, I looked for those accounts on my Dashboard, but cannot find them. Similar to **Bank Accounts**, **Credit Card Accounts**, and **Cash Accounts**, I expected them to show up under their own panel on the Dashboard called Investments (or something similar to that).

I looked in the previous discussions and Issues links here on the site but did not find anything about Investment accounts not showing up on the Dashboard.

Are they supposed to appear on the Dashboard like that?

Is it possible I hit the limit on the number of accounts that can be displayed (I have a lot)

Thank you.

**Comment by Ocirasai** [comment]
I did some further testing on this, and I can delete the **Investment** accounts I created, but I cannot see to reallocate or edit them.

I did try looking in the Help system, but I could not find anything on how Investment accounts are supposed to work or how you purchase shares to track.

Is there another resource elsewhere that contains this information?

Thank you.

**Reply by tactilis** [reply]
Does this help at all?

https://moneymanagerex.org/docs/features/stocks/

**Comment by Ocirasai** [comment]
Thank you for the link, but the problem I seem to be having is when I create an account of type **Investment**, I'm unable to see that on my Dashboard. do those accounts reside elsewhere?

Thank you.

**Comment by Ocirasai** [comment]
Just in case I misunderstood (easy to do at times) I re-read the help again:

***

1.1. Creating a Stock Portfolio for our Stocks

In order to use Stocks and Shares in MMEX, we need to create an account of type: Investment. 

The name for this account would generally be the name of the StockBroker. In our example we have selected the stock broker being called “Yahoo Finance”. 

This account would appear in the navigation tree under the heading Stock Portfolios. When we expand the Stock Portfolios branch in the navigation tree, we would see our newly created Stock Portfolio account. 

Selecting our new account “Yahoo Finance” opens the view: StockPortfolio: Yahoo Finance. In this view we can now add the stocks that we own.

***

In my case, I have an account at E*Trade Financial, so I selected this type of account to add.

<img width="230" alt="Investment Account" src="https://github.com/moneymanagerex/moneymanagerex/assets/140843655/be1d6e0b-3c64-449b-8fe5-8b55acdc33e7">

I then opened the Navigation Panel (I normally have this closed) and looked under Stock Portfolios like the help says, but I do not see this account listed:

<img width="286" alt="Stock Portfolios" src="https://github.com/moneymanagerex/moneymanagerex/assets/140843655/9410a015-f828-4931-9fd0-f304244bbe1b">

I also do not see this account on the Dashboard page at all under its own Panel for Investments (similar to Cash Accounts, Bank Accounts, etc.) which is where I thought it would appear at a minimum.

I also do not see a way to access it from the menu directly.

If I elect to Edit or Reallocate an account, it does not show up in the list of accounts that I can work with.

However, if I select Delete Account, it does show up there, so I believe the account is somewhere, I'm just not sure where.

Thank you.

**Comment by Ocirasai** [comment]
OK - So I think I figured out part of the issue.

I tried changing the theme I was using, and once I did, and restarted the application, the whole view in the Navigation Panel changed, and I could see that there was a ">" next to Stock Portfolios.

I suspect this might be a refresh issue and closing out of the program and then re-opening it redrew that section.

Is the only location I can see an Investment account on the Navigation Pane?

I would expect to be able to see all of my accounts on the Dashboard and I prefer the minimalist layout (No icons at the top and no Navigation Pane).

**Comment by Ocirasai** [comment]
OK - After adding my first set of shares to one of my Brokerage accounts using the Navigation Pane, I now see the Stocks Window on the Dashboard.

**Comment by tactilis** [comment]
> I suspect this might be a refresh issue and closing out of the program and then re-opening it redrew that section.

It was not an issue with the theme or a refresh issue.

You had the Navigation tree scrolled to the right so that the open/close arrows down the left had side were not visible:

![image](https://github.com/moneymanagerex/moneymanagerex/assets/4060092/c8e9b566-f978-47a4-b878-1a1ae450e1bb)

**Comment by Ocirasai** [comment, marked as answer]
My mistake - Thank you. :)
